FE-102/fe-102-browse-page.html

```diff
@@ -0,0 +1,293 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../FE-101/main.css" />
+    <link rel="stylesheet" href="browse-page.css" />
+    <title>Clacks & Thocc - Browse</title>
+  </head>
+  <body>
+    <nav class="nav">
+      <a class="title-logo" href="../FE-101/fe101-article-page.html">
+        Clacks&
+        <span>Thocc</span>
+      </a>
+      <img
+        class="nav-logo"
+        src="../FE-101/images/keyboard-logo.png"
+        alt="colored keyboard logo" />
+      <ul>
+        <li><a href="../FE-101/fe101-article-page.html">Home</a></li>
+        <li>
+          <a
+            href="https://www.codeandtheory.com/location/manila"
+            target="_blank">
+            About Us
+          </a>
+        </li>
+      </ul>
+    </nav>
+    <section class="grid-container">
+      <div class="opening-section">
+        <header>
+          <h1>Lorem Ipsum Dolor sit amet</h1>
+        </header>
+        <div class="featured-articles">
+          <div class="feature-article-card">
+            <a href="../FE-102/fe-102-detail-page-sample1.html">
+              <div class="img-box">
+                <img
+                  class="feature-article-img"
+                  src="../FE-101/images/featureImage-SizeDown.jpeg"
+                  alt="two keyboard sizes" />
+              </div>
```

Comment:
Instead of wrapping `img` in a `div`, we can wrap it in a `figure` tag instead for better semantic structure.

Please also check other instances.